tests/tasks/create_team_profile.yml

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- include_role:
+    name: linux-system-roles.network
+  vars:
+    network_connections:
+      - name: "{{ interface }}"
+        persistent_state: present
+        type: team
+        ip:
+          dhcp4: false
+          auto6: false
+
+      # add an team profile to the team
+      - name: member1
+        state: up
+        type: team
+        interface_name: team1
+        master: "{{ interface }}"
+
+      # add a second team profile to the team
+      - name: member2
+        state: up
+        type: team
+        interface_name: team2
+        master: "{{ interface }}"
```

Comment:
This is not needed. The test is creating just a team interface not attaching ports to it. Let's keep this simple and improve it in future work.